packages/skill-api/src/core/services/send-feedback-from-user.ts

```diff
@@ -42,17 +43,43 @@ export async function sendFeedbackFromUser({
       })
     }
 
-    const html = `${sanitizeHtml(feedbackText)} <i>${
-      context?.url ? context.url : ''
-    }</i>`
+    let from: string
```

Comment:
add conditionals to send the correct data for the email 

<img width="666" alt="Screenshot 2024-09-10 at 4 32 35 PM" src="https://github.com/user-attachments/assets/29de205c-0d12-419b-ab55-2cf95a206df0">